app/models/audio_file.rb

```diff
@@ -1,6 +1,17 @@
 class AudioFile < ApplicationRecord
   belongs_to :user
+  has_many :activities
```

Comment:
Would an audio file have many activities? I would imagine it would have one associated with it rather than many. This would apply to the other activity subjects as well I'd imagine.